tests/backend/test_create_user.py

```diff
@@ -0,0 +1,44 @@
+from fastapi.testclient import TestClient
+from fastapi import FastAPI
+import pytest
+from backend.main import app
```

Comment:
in backend's __init__.py have a 
``from .main import app``
Then change this import to ``from backend import app``